README.md

```diff
@@ -267,27 +267,35 @@ can be accomplished with a `RemoteChannel`:
 using ProgressMeter
 using Distributed
 
-p = Progress(10)
-channel = RemoteChannel(()->Channel{Bool}(10), 1)
+n_steps = 20
+p = Progress(n_steps)
+channel = RemoteChannel(()->Channel{Bool}(), 1)
```

Comment:
Note that the `Channel` has no buffer, so that `put!(channel, ...)` will block until the progress bar is updated; this way, the `yield()` is already included and does not need to be called separately.